site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -1,164 +1,134 @@
 
-      **Note:** The above path for all users was added in Chrome 16. Prior versions used a different
-      path:
-      `/Applications/Google Chrome.app/Contents/Extensions/` This path was deprecated in version 17.
-      Support was removed in version 20. Use one of the paths above instead.
+{% Aside %}
 
-      {% endAside %}
+**MacOS X and Linux only:** Do not use the preferences file for Windows. Use [Windows
+registry][section-registry] instead.
 
-    Linux:
+{% endAside %}
 
-    : `/opt/google/chrome/extensions/`
+### macOS {: #preference-mac }
 
-      `/usr/share/google-chrome/extensions/`
+1. Create a JSON file with the name of the extension ID. For example: `aaabbbcccdddeeefff.json`
+2. Place it in one of the folders listed below:
 
-      **Note:** Use `chmod` if necessary to make sure that the `aaaaaaaaaabbbbbbbbbbcccccccccc.json`
-      files are world-readable.
+    For a specific user
 
-3.  Linux only: If you are installing from a file, specify the extension's location and version with
-    fields named "external_crx" and "external_version" in the file created above.
-    - Example:
-    - ```json
-        {
-          "external_crx": "/home/share/extension.crx",
-          "external_version": "1.0"
-        }
-      ```
-    - **Note:** You need to escape each `\` character in the location. For example,
-      `\\server\share\extension.crx` would be `"\\\\server\\share\\extension.crx"`.
-4.  If you are installing from an update URL, specify the extension's update URL with field name
-    "external_update_url".
-5.  Example of installation from local .crx file (Linux only):
-    1.  ```json
-          {
-            "external_update_url": "http://myhost.com/mytestextension/updates.xml"
-          }
-        ```
-6.  Example of installation from the Chrome Webstore (Mac and Linux):
-    1.  ```json
-          {
-            "external_update_url": "https://clients2.google.com/service/update2/crx"
-          }
-        ```
-7.  If you would like to install extension only for some browser locales, you can list supported
-    locales in field name "supported_locale". Locale may specify parent locale like "en", in this
-    case the extension will be installed for all English locales like "en-US", "en-GB", etc. If
-    another browser locale is selected that is not supported by the extension, the external
-    extensions will be uninstalled. If "supported_locales" list is missing, the extension will be
-    installed for any locale.
-    - Example:
-    - ```json
-        {
-          "external_update_url": "https://clients2.google.com/service/update2/crx",
-          "supported_locales": [ "en", "fr", "de" ]
-        }
-      ```
-8.  Save the JSON file.
-9.  Launch Google Chrome and go to **chrome://extensions**; you should see the extension listed.
+    : `~USERNAME/Library/Application Support/Google/Chrome/External
+    Extensions/`
+
+    For all users
+
+    : `/Library/Application Support/Google/Chrome/External Extensions/`
```

Comment:
@rdcronin, is this path still accurate?

Reply:
As far as I know, but I don't know if I've explicitly tested this one.  Might be worth testing locally to verify.

Reply:
I tested this. It works 👍